File: crates/varde-code/tests/fixtures/php/domain_specific.php
<?php
// Domain-specific fixtures: Laravel-style Route (a `Route::<verb>` static call
// with a string path) and Response (`view`/`json`/`response()->json` — Laravel's
// response-producing helpers, body_shape set to the helper name).

Route::get('/users', 'UserController@index');
Route::post('/users', 'UserController@store');

function index()
{
    return view('users.index');
}

function store()
{
    return response()->json(['ok' => true]);
}

File: crates/varde-code/tests/fixtures/ruby/domain_specific.rb
# Domain-specific fixtures: Sinatra-style Route (an HTTP-verb call with a
# string path and a block) and Response (`json`/`erb`/`redirect` — Sinatra's
# response-producing helpers).

get "/users" do
  json([{ id: 1 }])
end

post "/users" do
  status 201
  json({ created: true })
end

get "/health" do
  erb :health
end

delete "/users/:id" do
  redirect "/users"
end
